### DISCUSSION TITLE
Restore backup in Kimai?

### DISCUSSION THREAD

**Discussion by BrunoAlm29** [kimai/kimai#3946] [On-Premise: Installation & Updates]
Hello,
Is there a way to restore deleted data on Kimai?

I went to delete an incorrectly created client, and when deleting it deleted all clients, projects, activities and all the activity of team members. It is possible?

Is there a way to rollback?

How can I solve this serious problem?

Thank you for your help.

BrunoA

**Comment by kevinpapst** [comment]
Self hosted? Do you have a backup?

**Comment by BrunoAlm29** [comment]
We have the Plus plan subscribe, from what I saw on the website we have
weekly backups, is that true?

But, something is not right on the platform. Because I just deleted a
client and it disappeared, everything.

Including departments. Only the users remained.

It doesn't make sense, something went wrong.

Need help.

**Reply by BrunoAlm29** [reply]
Everything has been erased, the platform is empty, only the users and the Tags remain.

Did all the work recorded for 2 years disappear because of the end of a client? I don't think I did anything wrong or different than what I've done several times to rectify duplicate customers.

**Reply by kevinpapst** [reply]
This is the wrong place, please use the Support Center at https://www.kimai.cloud/support-center/request

**Reply by kevinpapst** [reply]
If you don't have access, get in touch via company email at support@kimai.cloud
